app/models/spree/subscription.rb

```diff
@@ -127,6 +127,23 @@ def send_prior_notification
       end
     end
 
+    def cancel_stripe_subscription
+      raise "Stripe subscription ID not present. Subscription ID: #{stripe_subscription_id}" unless stripe_subscription_id.present?
```

Comment:
since stripe_subscription_id is already not there, so no point to append in string.